docs/about.md

```diff
@@ -14,13 +14,20 @@ The **project team**:
 * [**Aaron Chong Jun Hao**](https://github.com/acjh): _Project Mentor_ since Jul 2018, _Team Lead_ for Aug 2017 - Jun 2018
 * [**Chng Zhi Xuan**](https://github.com/Chng-Zhi-Xuan): _Project Member_ since May 2018
 * [**Chua Yun Zhi Nicholas**](https://github.com/nicholaschuayunzhi): _Project Member_ since Jan 2018
-* [**Damith C. Rajapakse**](https://github.com/damithch): _Project Mentor_ since Aug 2016
-* [**Daniel Berzin Chua Yuan Siang**](https://github.com/danielbrzn): _Project Member_  Jan 2018 - Aug 2018
+* [**Damith C. Rajapakse**](https://www.comp.nus.edu.sg/~damithch/): _Project Mentor_ since Aug 2016
 * [**Jamos Tay**](https://github.com/jamos-tay): _Project Member_ since Aug 2018
-* [**Jia Zhixin**](https://github.com/nusjzx): _Project Member_ May 2018 - Aug 2018
 * [**Jiang Sheng**](https://github.com/Gisonrg): _Founding Member_ and _Team Lead_ for Aug 2016 - Jul 2017
```

Comment:
@damithc should we update for Jiang Sheng as well?

Reply:
Yes, we can do that.